VideoWeb/VideoWeb/Helpers/AllocationHearingsEventNotifier.cs

```diff
@@ -21,19 +21,30 @@ public async Task PushAllocationHearingsEvent(UpdatedAllocationJusticeUserDto up
         {
             return;
         }
-
+    
         var conferences = (await conferenceService.GetConferences(conferenceIds)).ToList();
+        var usernamesToNotify = new List<string>();
         foreach (var conference in conferences)
         {
+            var previouslyAllocatedCsoUsername = conference.AllocatedCso;
+            var newAllocatedCsoUsername = update.AllocatedCsoUsername;
+            
+            if (!string.IsNullOrEmpty(previouslyAllocatedCsoUsername))
```

Comment:
You could remove the need to check by using the .distinct function on the list

Reply:
we still need to continue processing the conference though even if the previous username is null or empty - adding the new cso to the list, updating the conference etc. Or have I misunderstood

Reply:
oh sorry it's on me. what you're doing here is correct. I thought you were checking if the username was already tracked but noticed the loop at the bottom. If CSO A had 2 hearings reassigned from them to CSO B then CSO A needs to updates, 1 per conference/booking re-assigned?

Reply:
actually yes you make a good point there. If there are multiple conferences, we need to use a distinct list of usernames to notify in order to avoid pushing the update multiple times to each user.

have pushed a change to handle this. Also updated the test to cover multiple conferences